flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSourceTest.java

```diff
@@ -423,37 +418,38 @@ public void testConstructorParams() throws Exception {
             // connection fails but check if args have been passed correctly
         }
 
-        assertEquals("hostTest", testObj.getFactory().getHost());
-        assertEquals(999, testObj.getFactory().getPort());
-        assertEquals("userTest", testObj.getFactory().getUsername());
-        assertEquals("passTest", testObj.getFactory().getPassword());
+        assertThat(testObj.getFactory().getHost()).isEqualTo("hostTest");
+        assertThat(testObj.getFactory().getPort()).isEqualTo(999);
+        assertThat(testObj.getFactory().getUsername()).isEqualTo("userTest");
+        assertThat(testObj.getFactory().getPassword()).isEqualTo("passTest");
     }
 
-    @Test(timeout = 30000L)
-    public void testCustomIdentifiers() throws Exception {
+    @Test
+    @Timeout(30)
```

Comment:
```suggestion
    @Timeout(value = 30_000, unit = TimeUnit.MILLISECONDS)
```
I feel like adding the time unit here helps the readability. Especially, because readers might be more used to using milliseconds as a timeout unit. `@Timeout(30)` might leave the wrong impression. WDYT?